### PR TITLE
config metro in the example to have the Lib on runtime

### PR DESCRIPTION
**Pull request by benevbright**
#10 

I'm following this article
https://callstack.com/blog/adding-an-example-app-to-your-react-native-library/

But I'm getting this error. Work in progress (help please)
Screenshot 2019-12-29 at 13 15 03

### CODE REVIEW

**Comment by benevbright** [comment]
@hyochan Could you help this? 🙏

File: example/babel.config.js [hunk 1]
```diff
@@ -1,3 +1,13 @@
 module.exports = {
   presets: ['module:metro-react-native-babel-preset'],
+  plugins: [
+    [
+      'module-resolver',
+      {
+        alias: {
+          'react-native-toast-banner': '..',
```
Comment (hyochan) [inline example/babel.config.js:8]: Try `../index`

Comment (hyochan) [reply example/babel.config.js:8]: I am suspecting `package.json` might disturb this behavior since it is directing to built one.

Comment (benevbright) [reply example/babel.config.js:8]: 😭 no luck. I just tried that one and such like `../lib/index.js` as well but I get the same error.

**Comment by benevbright** [comment]
Found more references:
https://github.com/facebook/react-native/issues/21310#issuecomment-540227031
and
https://dev.to/brunolemos/tutorial-100-code-sharing-between-ios-android--web-using-react-native-web-andmonorepo-4pej

**Comment by benevbright** [comment]
@hyochan I found the solution :) https://github.com/benevbright/react-native-toast-banner/pull/14
Thanks for the help.

**Pull request closed without merging**